build_library/catalyst_toolchains.sh

```diff
@@ -16,10 +16,13 @@ configure_target_root() {
     local cross_chost=$(get_board_chost "$1")
     local profile=$(get_board_profile "${board}")
 
+    # Get the host CBUILD.
+    local cbuild="$(portageq envvar CBUILD)"
+
     CHOST="${cross_chost}" \
         ROOT="/build/${board}" \
         SYSROOT="/usr/${cross_chost}" \
-        _configure_sysroot "${profile}"
+        _configure_sysroot ${cbuild} "${profile}"
```

Comment:
It would be more in keeping with the existing (admittedly awkward) pattern to pass CBUILD via the environment. So something like:

```sh
CHOST="${cross_chost}" \
    CBUILD="$(portageq envvar CBUILD)" \
    ROOT="/build/${board}" \
    SYSROOT="/build/${board}" \
    _configure_sysroot "${profile}"
```

Any preferences? In this case _configure_sysroot wouldn't need to be modified.